test/testcafe/spec/ChallengeAuthenticatorPhone_spec.js

```diff
@@ -6,13 +6,29 @@ import phoneVerificationVoiceThenSMS from '../../../playground/mocks/data/idp/id
 import phoneVerificationVoiceOnly from '../../../playground/mocks/data/idp/idx/authenticator-verification-data-phone-voice-only';
 import smsVerification from '../../../playground/mocks/data/idp/idx/authenticator-verification-phone-sms';
 import voiceVerification from '../../../playground/mocks/data/idp/idx/authenticator-verification-phone-voice';
-import phoneVerificationSMSThenVoiceNoNumber from '../../../playground/mocks/data/idp/idx/authenticator-verification-data-phone-sms-then-voice-no-number';
-import phoneVerificationVoiceThenSMSNoNumber from '../../../playground/mocks/data/idp/idx/authenticator-verification-data-phone-voice-then-sms-no-number';
-import smsVerificationNoNumber from '../../../playground/mocks/data/idp/idx/authenticator-verification-phone-sms-no-number';
-import voiceVerificationNoNumber from '../../../playground/mocks/data/idp/idx/authenticator-verification-phone-voice-no-number';
+import phoneVerificationSMSThenVoiceNoProfile from '../../../playground/mocks/data/idp/idx/authenticator-verification-data-phone-sms-then-voice-no-profile';
+import phoneVerificationVoiceThenSMSNoProfile from '../../../playground/mocks/data/idp/idx/authenticator-verification-data-phone-voice-then-sms-no-profile';
+import smsVerificationNoProfile from '../../../playground/mocks/data/idp/idx/authenticator-verification-phone-sms-no-profile';
+import voiceVerificationNoProfile from '../../../playground/mocks/data/idp/idx/authenticator-verification-phone-voice-no-profile';
 import success from '../../../playground/mocks/data/idp/idx/success';
 import invalidCode from '../../../playground/mocks/data/idp/idx/error-email-verify';
 
+const phoneVerificationSMSThenVoiceNoNumber = JSON.parse(JSON.stringify(phoneVerificationSMSThenVoiceNoProfile));
```

Comment:
same nit on this file, I think it is better to name the variable with `EmptyProfile` suffix, to be more accurate